pkg/internal/common/persistence/connection.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package persistence
```

Comment:
i am wary of a common package holding persistence structures. These structures should be in their associated packages.

Reply:
this package doesn't maintain any structures, it is just like a util package. Isolating read, write, hashing to separate package, so that it will be easier to maintain in future.
As per the flow: service, restapi and client may need to access persistence (sometimes duplicate logic), so thought of moving it to separate package and using it as util from all layers.

may be package name is confusing?

Reply:
// ConnectionRecord takes care of connection related persistence
 type ConnectionRecord struct {
 	store storage.Store
 }

Reply:
gotcha, any package suggestion where it can be moved?

Reply:
fixed